### DISCUSSION TITLE
Hover off plugin

### DISCUSSION THREAD

**Discussion by temmiexva** [OpenTabletDriver/OpenTabletDriver#2096] [Plugins]
I have an idea to have a plugin. the plugin will disable hover and the tablet can only be used when the pen is pressed. This option is well suited to relative mode. (relative mod not working)

**Comment by Mrcubix** [comment]
It's called hover limiter and it's already available

**Reply by jamesbt365** [reply]
https://github.com/Kuuuube/Hover_Distance_Limiter

**Comment by temmiexva** [comment]
Thanks guys
